src/main/kotlin/org/tribot/gradle/plugin/TribotPlugin.kt

```diff
@@ -68,13 +68,13 @@ class TribotPlugin : Plugin<Project> {
                         .resolve("install")
                         .resolve("tribot-client")
                         .resolve("lib")
-                        .listFiles { dir, name -> name.matches("tribot-client-.*\\.jar".toRegex()) }
-                        ?.first()
+                        .listFiles { dir, name -> name.matches("legacy-client-.*\\.jar".toRegex()) }
+                        ?.firstOrNull()
                 if (tribotJar != null) {
                     it.dependencies.add("api", it.files(tribotJar))
                 }
                 else {
-                    it.dependencies.add("api", "org.tribot:tribot-client:+")
+                    it.dependencies.add("api", "org.tribot:legacy-client:+")
```

Comment:
This should actually still be `tribot-client` since it's the old maven repo jar

Reply:
Sorry boss - uno momento

Reply:
Done